cdap-api/src/main/java/co/cask/cdap/api/service/http/HttpServiceConfigurer.java

```diff
@@ -33,4 +34,14 @@
    * @param properties the properties to set
    */
   void setProperties(Map<String, String> properties);
+
+  /**
+   * Sets the {@link RetryPolicy} to use when a retry-able remote failure is encountered.
```

Comment:
from a quick google search, 'retryable' seems to be the more common spelling so I'll remove the dash. Apparently Spring has a Retryable annotation and python, javascript, ruby have projects called 'retryable'.